docs/openapi_v1.0.0-alpha.yaml

```diff
@@ -1162,6 +1211,17 @@ components:
       schema:
         type: string
         format: uuid
+    OptionIds:
+      name: optionIds
+      in: query
+      description: |
+        A list of valid option IDs that matches `id`s returned from `GET /products`. TODO: IS THIS REALLY CORRECT??
+      schema:
+        type: array
+        items:
+          type: string
+          format: string
+      required: false
```

Comment:
If the `OptionIds` are optional, I wonder if there is value in keeping them as part of the request parameters since the filtering could be done on the client side. Are we seeing performance gains with server-side filtering?

Reply:
Included your change

Reply:
@gimpster I've posted this one on #core_api.